pkg/controller/bootstrap/bootstrap.go

```diff
@@ -112,6 +114,8 @@ func (b *Bootstrap) Run(destDir string) error {
 				icspRules = append(icspRules, obj)
 			case *apicfgv1.Image:
 				imgCfg = obj
+			case *configv1.FeatureGate:
+				featureGate = obj
```

Comment:
IIRC, we only care when the feature gate has the correct name? Should we filter here?
```suggestion
				if obj.GetName() == ctrlcommon.ClusterFeatureInstanceName {
					featureGate = obj
				}
```